awx/settings/defaults.py

```diff
@@ -1090,6 +1090,7 @@
 CLUSTER_HOST_ID = socket.gethostname()
 
 UI_NEXT = True
+UI_LEGACY = False
```

Comment:
Why do we have two variables, rather than one variable which is set to the UI type (constant)

Reply:
yeah this change is meant to be additive in nature. There are handful of ui places that depend on UI_NEXT setting so remove this setting will require a bunch of other changes.

we have some followup work outlined that will clean this up though.

Reply:
it wasn't a lot of work to remove UI_NEXT setting, so I went ahead and remove it